src/main/java/com/nirima/jenkins/plugins/docker/action/DockerBuildImageAction.java

```diff
@@ -32,6 +33,7 @@
     public final boolean pushOnSuccess;
     public /* almost final */ boolean noCache;
     public /* almost final */ boolean pull;
+    public Map<String, String> labels;
```

Comment:
See [comment below](https://github.com/jenkinsci/docker-plugin/pull/769#discussion_r368930828) - I think it'd be better to store `labelString` rather than the parsed result of the string the user specified.
If we only store the parsed result, we can't store an invalid result, meaning that anything the user types in that we can't parse gets destroyed rather than letting the user go back and re-edit it.